pkg/routes/routes_test.go

```diff
@@ -20,13 +20,15 @@ import (
 )
 
 func TestPingdomEmpty(t *testing.T) {
+	// launch etcd server
 	e := setup(t)
 	defer e.Close()
+
 	etcD, err := storage.NewEtcdStorage([]string{"localhost:2379"})
 	assert.Nil(t, err)
```

Comment:
A lot of these tests use `assert`, however the code that follows should be continued. Using `require` to fail the test immediately would be the correct approach.